pkg/util/proc/monitoredproc_test.go

```diff
@@ -100,7 +99,6 @@ func TestMonitoredRestart(t *testing.T) {
 			assert.True(t, proc.monitor)
 			logger.Debugf("stop monitoring")
 			proc.monitor = false
-			break
```

Comment:
This raises a question for me. This reads more like C syntax than Go. @travisn or @jbw976 is the `case iter == 0:` above supposed to `fallthrough` to the `case iter == 1:` case?

Reply:
We must not need the fallthrough for iter 0 or else the test would be failing.